tests/e2e/cli-upgrade/chainsaw-test.yaml

```diff
@@ -103,10 +103,6 @@ spec:
       try:
         - apply:
             file: ../../common/apply/add-tempo-traces-destination.yaml
-    - name: Odigos pipeline ready
-      try:
-        - assert:
-            file:  ../../common/assert/pipeline-ready.yaml
```

Comment:
This guaranteed that we do not progress until the pipeline is ready.
Wonder if removing the assert can cause issues

Reply:
I added this step back, but only to make sure all the pods are ready, not asserting all the pods spec.
The full pipeline ready assert will be after the upgrade as before